public/components/DatasetPreview.vue

```diff
@@ -13,7 +13,10 @@
         <i class="fa fa-table"></i> <b>Dateset Name:</b>
         {{ dataset.name }}
       </a>
-      <a class="nav-link"><b>Features:</b> {{ dataset.variables.length }}</a>
+      <a class="nav-link"
+        ><b>Features:</b>
+        {{ dataset.variables.filter(v => v.distilRole === "data").length }}</a
```

Comment:
Usually, we create a **computed** function to not bloat the HTML with code. But I don't think it's an issue here.